src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -47,7 +47,7 @@ public bool ExecuteCommand()
             var extension = Path.GetExtension(InputFilePath);
             if (string.Equals(extension, ".csproj", StringComparison.OrdinalIgnoreCase))
             {
-                if (PlatformHelper.IsMono)
+                if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
MSBuild exists on windows and since it shells out it will work with CoreCLR on windows.